src/sc_crawler/vendors/_gcp.py

```
@@ -1391,19 +1408,54 @@ def inventory_databases(vendor):
+                "storage_extra_autosize": True,
+                # https://cloud.google.com/sql/docs/postgres/cmek
+                "disk_encryption": True,
+                # Minor versions are applied via Cloud SQL maintenance updates.
+                # https://cloud.google.com/sql/docs/postgres/maintenance
+                "auto_upgrade_versions": True,
                 # https://cloud.google.com/sql/docs/postgres/backup-recovery/backups
                 "scheduled_backups": True,
-                # https://cloud.google.com/sql/docs/postgres/backup-recovery/pitr
-                "continuous_backups": None,
+                # Max transactionLogRetentionDays for PITR by edition; not in tiers API.
+                # Enterprise: 1–7 days; Enterprise Plus: 1–35 days.
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)